tests/ImageSharp.Web.Tests/Commands/CommandParserTests.cs

```diff
@@ -75,10 +75,12 @@ public class CommandParserTests
 
         public static TheoryData<object, string> Rgba32Values = new TheoryData<object, string>
         {
-            { Rgba32.White, "255,255,255" },
-            { Rgba32.Transparent, "255,255,255,0" },
-            { Rgba32.Orange, "orange" },
-            { Rgba32.Lime, "00FF00" },
+            { Color.White, "255,255,255" },
+            { Color.Transparent, "0,0,0,0" },
```

Comment:
note that i have changed the expected value from transparent from 255, 255, 255,0 to 0,0,0,0 as i think that's the correct value for transparent, see: [transparent](https://www.w3.org/TR/css-color-3/#transparent-def)

Reply:
Yep, that's correct. W3C defines transparent as 0,0,0,0 only MS use 255,255,255,0